samples/go/csv/read_test.go

```diff
@@ -133,6 +133,28 @@ g,h,i,j
 `
 	testTrailingHelper(t, dataString)
 }
+func TestEscapeStructFieldFromCSV(t *testing.T) {
+	assert := assert.New(t)
+	cases := []string{
+		"a", "a",
+		"1a", "a",
+		"AaZz19_", "AaZz19_",
+		"Q", "Q",
+		"AQ", "AQ",
+		"$", "",
```

Comment:
I don't think we can have an empty string for a struct field name. If you end up stripping out all of the characters, what should happen?  It shouldn't be a valid struct field, so maybe it should panic? This and the emoji case below are interesting test cases that should probably either return an error or panic from EscapeStructFieldFromCsv.